protocols/kad/src/protocol.rs

```diff
@@ -35,6 +35,7 @@ use futures::prelude::*;
 use instant::Instant;
 use libp2p_core::upgrade::{InboundUpgrade, OutboundUpgrade, UpgradeInfo};
 use libp2p_core::{Multiaddr, PeerId};
+use log::debug;
```

Comment:
Personal preference only but I like not importing these because they often come and go with refactorings and `log::debug` is actually very expressive.